Userland/Applications/FileManager/main.cpp

```diff
@@ -806,6 +806,18 @@ ErrorOr<int> run_in_windowed_mode(DeprecatedString const& initial_location, Depr
         window);
     copy_action->set_enabled(false);
 
+    auto copy_path_action = GUI::Action::create(
+        "Copy Path", [&](GUI::Action const&) {
+            auto paths = directory_view->selected_file_paths();
+            if (paths.is_empty()) {
+                paths = tree_view_selected_file_paths();
```

Comment:
This is a bug: if you select both an entry in the tree view and the directory view, and then select Copy Path from the treeview, you get the path for the entry in the directory view instead.

Reply:
I have fixed this bug, and now using `is_focused()` to get selected file paths.